integration/configtx/configtx_test.go

```diff
@@ -115,8 +114,9 @@ var _ = Describe("ConfigTx", func() {
 		}
 
 		channelID := "testchannel"
-		createChannelUpdate, err := configtx.NewCreateChannelTx(channel, channelID)
+		createChannelUpdate, err := configtx.NewMarshaledCreateChannelTx(channel, channelID)
```

Comment:
This seems to be associated with the previous commit, right? If we bump the dependency without this change, will the code build? If not, this should be pulled into the previous commit.

Our CI (for better or worse) does not build and test each commit independently so it's easy to have a broken commit that could bite us during bisects or other postmortem activities.

Reply:
Good to know. Squashed the two commits together.